stable/chartmuseum/templates/deployment.yaml

```diff
@@ -91,13 +107,29 @@ spec:
     {{- end }}
 
       volumes:
+      {{- if (eq .Values.env.open.STORAGE "local") }}
```

Comment:
nice catch here,  but I think that the template will now break unless storage is local or openstack due to dangling `volumes:` key

Reply:
Indeed.
Not super elegant but I added a big if above and guards for gcp as well (not necessary but...) :)